modules/ROOT/pages/utility-classes-cookbook.adoc

```diff
@@ -74,7 +71,7 @@ To invoke static methods using DataWeave you must add the `java!` prefix to the
 </dependency>
 ----
 . Use any of the Java Module's operations. +
-In this case, invoke a static method of a class:
+For this example, invoke a static method of a class:
 +
 .Using a class with the Java Module
```

Comment:
If Java Module is a proper noun, remove "the."
If not, change to "the Java module."

Reply:
I've looked this up on the internet many times, and been in discussions with George and PMMs. No one seems to agree. Both uses are common. So if you have any advice or clarity about this, I'd sure love to hear it!